src/main/java/net/dv8tion/jda/core/audio/AudioConnection.java

```diff
@@ -174,9 +169,9 @@ public void setChannel(VoiceChannel channel)
         this.channel = channel;
     }
 
-    public JDA getJDA()
+    public JDAImpl getJDA()
     {
-        return channel.getJDA();
+        return (JDAImpl) channel.getJDA();
```

Comment:
Can't this be replaced with just `return api;`? (see line 91)

Reply:
No it can't be replaced with that, the api reference is local in the constructor.